src/segment-loader.js

```diff
@@ -635,6 +635,31 @@ export default class SegmentLoader extends videojs.EventTarget {
       this.roundTrip = request.roundTripTime;
       this.bandwidth = request.bandwidth;
       this.bytesReceived += request.bytesReceived || 0;
+      this.trigger({
+        type: 'stat',
+        data: {
+          name: 'numberOfBytesTransferred',
+          amount: request.bytesReceived || 0
+        }
+      });
+
+      // if we get this far there are no errors,
+      // increment the stat for media requests
+      this.trigger({
+        type: 'stat',
+        data: {
+          name: 'numberOfMediaRequests',
```

Comment:
Our standard pattern in video.js has been to fire "simple events" when interesting state changes and provide accessors to query state. For consistency, I think we should continue to follow that pattern here.